databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -232,7 +212,7 @@ def _create_indices(self, node_file: str) -> None:
         LOGGER.info('Creating indices. (Existing indices will be ignored)')
 
         with open(node_file, 'r', encoding='utf8') as node_csv:
-            for node_record in csv.DictReader(node_csv):
+            for node_record in pandas.read_csv(node_csv).to_dict(orient='records'):
```

Comment:
Record field types is string when using `csv.DictReader` which is inadequate when switching to use parameter. 

`pandas` has ability to infer data type when reading csv out. The type info can be useful for cypher parameter.